shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -10,7 +11,11 @@ pub fn cassandra_connection(contact_points: &str, port: u16) -> Session {
     cluster.set_credentials("cassandra", "cassandra").unwrap();
     cluster.set_port(port).ok();
     cluster.set_load_balance_round_robin();
-    cluster.connect().unwrap()
+    let result = cluster.connect();
```

Comment:
Isnt this equivalent to the previous implementation?

Reply:
I tried it out locally and I see what you mean.
While the relevant error is in there somewhere its hidden by A LOT of noise.

This change is likely to be accidentally drive-by "cleaned up" next time someone see's it as its not obvious why its needed.
So lets add a comment explaining whats going on and simplify it a little:
```rust
    // By default unwrap uses the Debug formatter `{:?}` which is extremely noisy for the error type returned by `connect()`.
    // So we instead force the Display formatter `{}` on the error.
    cluster.connect().map_err(|err| format!("{err}")).unwrap()
```